scipy/interpolate/ndgriddata.py

```diff
@@ -41,8 +41,11 @@ class NearestNDInterpolator(NDInterpolatorBase):
         Rescale points to unit cube before performing interpolation.
         This is useful if some of the input dimensions have
         incommensurable units and differ by many orders of magnitude.
-
         .. versionadded:: 0.14.0
+    cKDTree_options : dict, optional
```

Comment:
I typically shun away from bikeshedding names, but this one is really suboptimal, there's no way I can type right from the first try. `tree_options` or just `options`?